pages/linux/pwn.md

```diff
@@ -0,0 +1,36 @@
+# pwn
+
+> Exploit Development Library designed for rapid prototyping.
+> More information: <https://docs.pwntools.com/en/stable/commandline.html>.
+
+- Convert  the given assembly code to `bytes`:
+
+`pwn asm "{{xor edi, edi}}"`
+
+- Create a cyclic pattern of 100 characters:
+
+`pwn cyclic {{100}}`
```

Comment:
```suggestion
- Create a cyclic pattern of the specific number of characters:

`pwn cyclic {{value}}`
```

Reply:
I guess having {{value}} sounds more generic IMO since only numbers could be mentioned.